src/user/organization/organization.js

```diff
@@ -53,121 +54,182 @@ class Organization extends Component {
             ) : (
-              <div className="posts">
-                <div className="categories">
-                  <div className="ul__container">
-                    <span className="nav__tab container">
-                      <ul className="nav__list__container">
-                        <li
-                          className={
-                            type === "About"
-                              ? "nav__single__tab selected"
-                              : "nav__single__tab"
-                          }
-                          onClick={this.handleClick.bind(this, "About")}
-                        >
-                          Overview
-                        </li>
-                        {/* <li
-                          className={
-                            type === "Post"
-                              ? "nav__single__tab selected"
-                              : "nav__single__tab"
-                          }
-                          onClick={this.handleClick.bind(this, "Post")}
-                        >
-                          Posts
-                        </li>
-                        <li
-                          className={
-                            type === "Event"
-                              ? "nav__single__tab selected"
-                              : "nav__single__tab"
-                          }
-                          onClick={this.handleClick.bind(this, "Event")}
-                        >
-                          Events
-                        </li>
-                        <li
-                          className={
-                            type === "Project"
-                              ? "nav__single__tab selected"
-                              : "nav__single__tab"
-                          }
-                          onClick={this.handleClick.bind(this, "Project")}
-                        >
-                          Projects
-                        </li> */}
-                      </ul>
-                    </span>
-                  </div>
-                </div>
-                <Card className="about-us">
-                  <CardContent>
-                    <div className="title">{orgProfile?.name}</div>
-                    <div className="subtitle">Short description </div>
-                    <p className="short__desc">
-                      {description?.shortDescription ||
-                        "Short details of organization"}
-                    </p>
-                    <div className="subtitle">About us in details</div>
-                    <p className="long__desc">
-                      {description?.longDescription ||
-                        "Long description of the organization"}
-                    </p>
-                  </CardContent>
-                </Card>
+              <div className="notify-user">
+                <OrgInfo></OrgInfo>
+                <Desktop>
+                  <Portfolio></Portfolio>
```

Comment:
Maybe use self-closing tags if nothing to be put inside

Reply:
Alright, I'll rectify though it was already there and I was told to just focus on responsiveness.

Reply:
Yeah! No issues, just you can refer these comments later on